examples/first-project/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample first project
+
+This is a sample project with a single contract for getting started with ZeppelinOS. Head over to our [documentation](https://docs.zeppelinos.org) to see the guide for this project.
+
+To run it on your machine:
+- Install dependencies with `npm install`
+- Start a ganache instance with `npx ganache-cli -p 9545 -d`
```

Comment:
The `network.js` file inside the `first-project` folder has the local network listening to 8545.